### PR TITLE
Test service client with net472 and net451

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: iothub/service/tests/ClientWebSocketTransportTests.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```
Comment (barustum) [inline iothub/service/tests/ClientWebSocketTransportTests.cs]: duplicates

File: iothub/service/tests/ClientWebSocketTransportTests.cs [hunk 2]
```diff
@@ -79,6 +79,7 @@ public async Task ClientWebSocketTransportReadWithoutConnectTest()
 
         // The following tests can only be run in Administrator mode
         [TestMethod]
+        [Timeout(10000)] // if it is going to fail, fail fast. Otherwise it can go on for 4+ minutes. :(
```
Comment (vinagesh) [inline iothub/service/tests/ClientWebSocketTransportTests.cs]: Just curious - why does this fail?

Comment (drwill-ms) [reply iothub/service/tests/ClientWebSocketTransportTests.cs]: The comment on 80 says it has to be run in admin mode.

In practice, I find it fails if run with other tests and not in admin mode. If you run it by itself, even not in admin mode, it'll succeed.

Comment (abhipsaMisra) [inline iothub/service/tests/ClientWebSocketTransportTests.cs]: nit: save this timeout value as a constant.

Comment (drwill-ms) [reply iothub/service/tests/ClientWebSocketTransportTests.cs]: Why?

Comment (abhipsaMisra) [reply iothub/service/tests/ClientWebSocketTransportTests.cs]: So that we don't end up having to maintain different values everywhere

Comment (drwill-ms) [reply iothub/service/tests/ClientWebSocketTransportTests.cs]: General rule of thumb is to make something a constant if it is referenced from more than one location. In this case, we aren't. The value only matters here. If I found a second method that needed the same value, then I'd consider it.

Comment (drwill-ms) [reply iothub/service/tests/ClientWebSocketTransportTests.cs]: I'm going to resolve this as by design, for now. Hit me up on teams if you want to talk about it. :)

**Pull request merged**